driver/pelican/main.go

```diff
@@ -39,7 +51,12 @@ func main() {
 	password := params.MustString("password")
 	sitename := params.MustString("sitename")
 
-	pelicans, err := DiscoverPelicans(username, password, sitename)
+	timezone, zoneErr := GetTimeZone(sitename, username, password)
```

Comment:
Let's fold the timezone logic into the `DiscoverPelicans` function.

So we don't provide the timezone as an argument to `DiscoverPelicans`, instead it makes the API call to figure out the site's timezone and populates the pelican structs that are returned with that value.